src/services/external/excalidraw-backend/excalidraw.server.ts

```diff
@@ -508,6 +515,24 @@ export class ExcalidrawServer {
       );
     }
   }
+
+  private async processActivityWhiteboardContentModified(
+    agentInfo: AgentInfo,
+    whiteboardId: string
+  ) {
+    const whiteboard = await this.whiteboardService.getWhiteboardOrFail(
+      whiteboardId,
+      {
+        relations: {
+          profile: true,
+        },
+      }
+    );
+    this.activityAdapter.calloutWhiteboardContentModified({
+      triggeredBy: agentInfo.userID,
+      whiteboard: whiteboard,
```

Comment:
pass the ID instead of the whole object. I would understand if this method asked for a whiteboard with it's profile loaded, but it does not. Only the person writing the code knows it.
Let the implementation decide what data to load, you just pass the constant ID.